### DISCUSSION TITLE
RIP Harpy :(

### DISCUSSION THREAD

**Discussion by dnestico** [robertodoering/harpy#882] [General]
Welp it was fun while it lasted and this was my favorite twitter client, unfortunately, twitter has now banned all 3rd-party clients. :(

Gonna continue using it as long as it works.

so sorry @robertodoering 

https://9to5google.com/2023/01/19/twitter-clients-banned/

**Comment by nyabinary** [comment]
Perhaps this could turn into a fediverse client?

**Reply by dnestico** [reply]
I would love a harpy Reddit client, but for now, it still seems to be working for twitter, really hoping Elon changes his mind but highly doubt it...

**Comment by robertodoering** [comment]
Well.. fuck.. this isn't the news I was hoping for :(

I'll create a separate post soon. RIP
